Package.swift

```diff
@@ -153,7 +153,7 @@ let package = Package(
             targets: ["Realm", "RealmSwift"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/realm/realm-core.git", exact: coreVersion)
+        .package(url: "https://github.com/realm/realm-core.git", revision: "v\(coreVersion)")
```

Comment:
This will make it so that the package cannot be imported. SPM has some very strange rules around non-version-based dependencies.

Reply:
No, a version tag is a hard requirement. We sometimes temporarily set it to a branch or other thing for local development or for running CI tests on PRs, but that has to be reverted before a release.

Reply:
Okay, I read the docs and it appears we always have to release with a tagged version of Core.